style/index.js

```diff
@@ -207,7 +207,11 @@ function generateFreemapStyle({
       .area(colors.allotments, 'allotments')
       .area(hsl(0, 0, 80), 'industrial', 'wastewater_plant')
       .area(hsl(320, 40, 85), 'commercial', 'retail')
-      .area(colors.wetland, 'wetland')
+      .area(colors.grassy, 'wetland')
```

Comment:
toto mazem lebo to renderuje wetland vsade. prosim pred pull-requestom otestuj ci to funguje